pycaret/internal/pycaret_experiment/tabular_experiment.py

```diff
@@ -357,7 +357,8 @@ def _initialize_setup(
             if cuml_version is None or not version.parse(cuml_version) >= version.parse(
                 "22.10"
             ):
-                message = f"cuML is outdated or not found. Required version is >=22.10, got {__version__}"
+                message = """cuML is outdated or not found. Required version is >=22.10.
```

Comment:
Windows can´t use cuml package, so we need to do someting about this: Windows should not have this message!
I created a PR to fix warnings about cuml package on windows.
PR: https://github.com/pycaret/pycaret/pull/3881